module/zfs/zfs_znode.c

```diff
@@ -90,6 +90,7 @@
 static kmem_cache_t *znode_cache = NULL;
 static kmem_cache_t *znode_hold_cache = NULL;
 unsigned int zfs_object_mutex_size = ZFS_OBJ_MTX_SZ;
+unsigned long zfs_delete_blocks = DMU_MAX_DELETEBLKCNT;
```

Comment:
Could you also fix the comment in dmu.h for DMU_MAX_DELETEBLKCNT, it's actually around 2.5MB (20 indirect blocks).

Reply:
Fixed.